guide/src/class/numeric.md

```diff
@@ -233,8 +233,9 @@ impl Number {
         Self(value)
     }
 
-    fn __repr__(&self) -> String {
-        format!("Number({})", self.0)
+    fn __repr__(slf: &PyCell<Self>) -> PyResult<String> {
+        let class_name: &str = slf.get_type().name()?;
```

Comment:
I think a comment here would be helpful, in case the user hasn't read the preceding chapter
 ```suggestion
        // Get the class name dynamically in case `Number` is subclassed
        let class_name: &str = slf.get_type().name()?;
```